src/LinkedIn.php

```diff
@@ -277,7 +277,7 @@ protected function getUserFromAccessToken()
         try {
             return $this->api('GET', '/v1/people/~:(id,firstName,lastName,headline)');
         } catch (LinkedInApiException $e) {
-            return;
+            return null;
```

Comment:
To show what you're returning is a best practice. If you are not agree with me, I can erase them.